main.sh

```diff
@@ -107,40 +99,40 @@ function xml_dump {
 
 function all_sch { # arg($1:group)
     # is loaded by default so data must be loaded
-    data=($(sed 's/://g' $SCHEDULE))
```

Comment:
But using that alias won't allow using options I think. Anyway, is 80x24 size requirement still relevant today? I don't think so.